src/resources/js/v2/tickets-loader.js

```diff
@@ -0,0 +1,93 @@
+/* global tribe */
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.tickets = tribe.tickets || {};
+
+/**
+ * Configures ET Loader Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.tickets.loader = {};
+
+/**
+ * Initializes in a Strict env the code that manages the plugin "loader".
+ *
+ * @since TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} obj tribe.tickets.loader
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	const $document = $( document );
+
+	/**
+	 * Selectors used for configuration and setup.
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		loader: '.tribe-common-c-loader',
+		hiddenElement: '.tribe-common-a11y-hidden',
+	};
+
+	/**
+	 * Show loader for the container.
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container jQuery object of the container.
+	 *
+	 * @return {void}
+	 */
+	obj.show = function( $container ) {
+		const $loader = $container.find( obj.selectors.loader );
+
+		if ( $loader.length ) {
+			$loader.removeClass( obj.selectors.hiddenElement.className() );
+		}
+	};
+
+	/**
+	 * Hide loader for the container.
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container jQuery object of the container.
+	 *
+	 * @return {void}
+	 */
+	obj.hide = function( $container ) {
+		const $loader = $container.find( obj.selectors.loader );
+
+		if ( $loader.length ) {
+			$loader.addClass( obj.selectors.hiddenElement.className() );
+		}
+	};
+
+	/**
+	 * Handles the initialization of the scripts when Document is ready.
+	 *
+	 * @since TBD
+	 *
+	 * @return {void}
+	 */
+	obj.ready = function() {
+		// Silence is golden.
+	};
+
+	// Configure on document ready.
+	$document.ready( obj.ready );
```

Comment:
we don't' need to run empty functions if we don't need it.

Reply:
I'm pretty sure this is code that Juan will be working on. But this would be a good candidate for a `@todo` marker explaining your intent of what you need to do. The function has that, but it's not clear that it wasn't done. Leaving `@todo` comments is super helpful to look back on as a dev working on stuff -- but even more helpful for everyone else :)